spec/takeaway_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rspec'
+require './lib/takeaway.rb'
+
```

Comment:
When running rspec it requires the user to actually manually run the program, and seems to miss some tests if you just quit instead of going through it - I'm guessing this is down to having to test an interactive test, so there's probably some very complex rspec syntax for stubbing that somewhere...god knows how though

Reply:
I had a look at this this morning, the reason that it requires the user to interact with the program when you run rspec is because I have called the method in the main program that runs the user-interaction loop - if I remove that then rspec just runs. Agree on the conditionals though, I'm not yet sure how you would satisfy yourself that all of the branches of the conditional statements were thoroughly tested. (the include 'rspec' isn't associated with the fact that you have to interact with the program when you run rspec - for some reason my terminal sometimes doesn't recognise rspec commands until I include this line).